### ISSUE TITLE
Unable to connect to Client at host

### ISSUE THREAD

**Issue by giejay** [d8ahazard/Phlex#132]
Looked through the closed issues and stumbled upon some issues which were related but all seemed to be fixed by some code updates. 
Im running my Phlex in a docker container on my raspberry pi (modified the docker image to use the arm64 base instead of amd64). 

Can reach my public address just fine from 4G and my internal network. Filled in that address in the GUI but whenever Im trying to link with Google, Im getting the unable to connect.

The weird thing is that I tried to link it with Amazon, and that worked fine... 

I am using http atm, also tried to use https but that gave the same results. Anyone an idea?

**Comment by silasrp** [comment]
Same thing here. I just find amazing how I problem that is so common through the forum doesn't have a definitive solution yet. I followed the procedures to install the application exactly as stated in the git documentation but still two things don't work:
- Always get the message "Unable to connect to Client at http://xxxxx:80/Phlex, please check the setting for 'Public Address' in the Phlex UI."
- RESCAN DEVICES don't find any device. I would expect it to find my Chromecast in the network.

**Comment by d8ahazard** [comment]
@silasrp Try now.  I took the method I was using for Amazon verification, which is a modified version of the Google one, and replaced it.  I'm able to link using http://IP:PORT/Phlex, so it should work.

Hopefully, if this gets you working, you can be as enthusiastic in reviewing the "Donations" section in the readme as you were in yelling about how amazing it is that I haven't found the time to work on this until now...

Y'know, considering as how this is the product of thousands of hours of my free time over the past nine months or so...

**Comment by bcb2000** [comment]
I also am getting the error "Unable to connect to Client at host ..." when trying to link with Google.

My set-up is on a Raspberry Pi, done by following the instructions in the wiki.  In addition to those instructions, I have an Apache forwarding proxy which performs name-based virtual hosting, then Proxy-Passes on to my RPi.  My Forward Proxy is ssl, and has a valid LetsEncrypt certificate for my phlex server.

I can connect to the my phlex URL externally, and If I trace my webserver's access logs, I can see that the registration server is accessing my server and getting a 200 response code:

```
==> /var/log/apache2/phlex-access.log <==
192.99.197.11 - - [20/Oct/2017:06:02:13 +1100] "GET /api.php?testclient&apiToken=redacted HTTP/1.1" 200 4391 "-" "-"
```
Furthermore, if I try to access the testclient url that 192.99.197.11 is going for, the response I get is "success"

I've tried various things to troubleshoot fix this:
 - I've moved the Phlex files in and out of the webserver root and the /Phlex/ subdirectory.  My preference is to have them in the webserver root, but for now I've left them back where they were by default, in /Phlex/
 - I've tried performing the activation from inside or outside my network

Any more troubleshooting suggestions?

Thanks

**Comment by bcb2000** [comment]
Further troubleshooting I've tried... I allowed access unsecured on port 80 and tried linking with google using 
- http://my.domain/Phlex/
- http://my.domain:80/Phlex/
- http://my.public.ip/Phlex/
- http://my.public.ip:80/Phlex/

In all cases, I still get the above behaviour.
And in all cases, the Amazon link works correctly.

Any suggestions?
Thanks

**Comment by bcb2000** [comment]
Solved it - see thread #162
